app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -232,7 +231,7 @@ class BrowserFragment : IWebViewLifecycleFragment(),
     private fun handleSpecialKeyEvent(event: KeyEvent): Boolean {
         if (event.keyCode == KeyEvent.KEYCODE_MENU && event.action == KeyEvent.ACTION_UP) {
             val toShow = !browserOverlay.isVisible
-            showOverlay(toShow)
```

Comment:
I didn't think there was a difference between these two so it would be fine to swap them, but what about https://github.com/mozilla-mobile/firefox-tv/commit/8747c4f4236c995e66348745d71e05acb9ce1691 ? Should that have fixed that bug, if they're the same, minus telemetry?

Reply:
That bug is irrelevant – the problem in that bug is that you can call 1) BrowserFragment's hide or 2) BrowserOverlay's hide. Ideally, the code could be architected differently so you can't do that.

What I did in this bug wasn't swap these implementations, but rename BrowserFragment's to be the same as BrowserOverlay's so it's more apparent that maybe you'd want to use it instead of touching the overlay directly and to make it clear that this should only be called from a user action (i.e. so telemetry isn't messed up). I'll add a comment.